__tests__/username-validation.ts

```diff
@@ -0,0 +1,73 @@
+import { UsernameValidation } from "../app/utils/validation"
+
+it("empty", () => {
```

Comment:
please use another description, I think is a general request for the tests in this file